### ISSUE TITLE
Set choice options for yes/no fields

### ISSUE THREAD

**Issue by AHelms** [fulcrumapp/fulcrum-expressions#30]
Currently, only single/multiple choice field and classification set fields can have their choice options set.

Add the ability to set the choice options for yes/no fields should allow users to edit all choice options dynamically as needed.

**Comment by bmcbride** [comment]
I'm having trouble understanding how dynamic yes/no options would be used... It also seems a little dangerous to dynamically set values for a boolean field.

**Comment by AHelms** [comment]
Yeah, you are correct. This is more for changing the labels and not the values of the choice options. I updated the original issue to better reflect this. 

The use case revolves around dynamically translating the app into different languages.
